components/filter/events.md

```diff
@@ -12,12 +12,52 @@ position: 11
 
 This article explains the available events for the Telerik Filter for Blazor:
 
+* [OnUpdate](#onupdate)
 * [ValueChanged](#valuechanged)
 
+## OnUpdate
+
+The `OnUpdate` event fires when the filter value changes. The component works directly with the bound `CompositeFilterDescriptor` value and automatically updates it when it is used with one-way binding.
```

Comment:
It's not very clear what is the benefit and use case of this event, especially when we have `ValueChanged`. Please elaborate.